io/plugins/eu.esdihumboldt.hale.io.xls.test/src/eu/esdihumboldt/hale/io/xls/test/writer/XLSInstanceWriterTest.java

```diff
@@ -209,6 +208,49 @@ public void testWriteNotNestedProperties() throws Exception {
 		checkFirstDataRow(sheet, firstDataRow);
 	}
 
+	/**
+	 * Test - write data of complex schema and analyze result
+	 * 
+	 * @throws Exception , if an error occurs
+	 */
+	@Test
+	public void testExportMultiFeatureToExcel() throws Exception {
+		ArrayList<InstanceCollection> examples = new ArrayList<>();
+		TransformationExample example = TransformationExamples
+				.getExample(TransformationExamples.SIMPLE_COMPLEX);
+		TransformationExample example2 = TransformationExamples
+				.getExample(TransformationExamples.CARDINALITY_MOVE);
+		examples.add(example.getTargetInstances());
+		examples.add(example2.getTargetInstances());
+
+		// set instances to xls instance writer
+		XLSInstanceWriter writer = new XLSInstanceWriter();
+		IContentType contentType = HalePlatform.getContentTypeManager()
+				.getContentType("eu.esdihumboldt.hale.io.xls.xls");
+		writer.setParameter(InstanceTableIOConstants.SOLVE_NESTED_PROPERTIES, Value.of(false));
+		writer.setParameter(InstanceTableIOConstants.USE_SCHEMA, Value.of(false));
+		writer.setParameter(InstanceTableIOConstants.EXPORT_EMPTY_FEATURETYPES, Value.of(true));
+
+		File tmpFile = tmpFolder.newFile("excelWith2Sheets.xls");
+		// write instances to a temporary XLS file
+		writer.setTarget(new FileIOSupplier(tmpFile));
+		writer.setContentType(contentType);
+
+		for (InstanceCollection instanceCollection : examples) {
+			writer.setInstances(instanceCollection);
```

Comment:
`setInstances` in this case goes back to this implementation: https://github.com/halestudio/hale/blob/5d335696fd7b8f7e2114d3423ea5d2cacf2445f2/common/plugins/eu.esdihumboldt.hale.common.instance/src/eu/esdihumboldt/hale/common/instance/io/impl/AbstractInstanceWriter.java#L88

By setting it multiple times you just replace what was set before.

Instead you would need to use something like this to create a combined instance collection:
https://github.com/halestudio/hale/blob/5d335696fd7b8f7e2114d3423ea5d2cacf2445f2/common/plugins/eu.esdihumboldt.hale.common.instance/src/eu/esdihumboldt/hale/common/instance/model/impl/MultiInstanceCollection.java#L42